instrumentation/spring/spring-rmi-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/rmi/v4_0/server/ServerInstrumentation.java

```diff
@@ -83,7 +83,7 @@ public static void stopSpan(
         return;
       }
       scope.close();
-      serverInstrumenter().end(context, request, null, throwable);
+      SpringRmiSingletons.serverInstrumenter().end(context, request, null, throwable);
```

Comment:
guessing Intellij did this?

Reply:
Yup, I didn't notice that.
I'll fix that everywhere, thanks!